### PR TITLE
Fixes #4287 - Inconsistency with notices in header

### LINKED ISSUE #4287: Inconsistency with notices in header
For example, the awaiting moderation header will show like such when you hover over it:

Awaiting Moderation: 1 unapproved thread.

The group leader notice will show like this when you hover over it:

Group Leader Notice: You have 1 pending group membership join request.

Original thread: [Inconsistency with notices in header](https://community.mybb.com/thread-231493.html)

### PR DESCRIPTION
**Pull request by Eldenroot**
Resolves #4287

### CODE REVIEW

File: global.php [hunk 1]
```diff
@@ -647,8 +647,7 @@
 			}
 			else
 			{
-				$total_joinrequests = my_number_format($total_joinrequests);
-				$lang->pending_joinrequests = $lang->sprintf($lang->pending_joinrequests, $total_joinrequests);
+				$lang->pending_joinrequests = $lang->sprintf($lang->pending_joinrequests, my_number_format($total_joinrequests));
```
Comment (Sama34) [inline global.php:650]: What is the reasoning for this change?

**Review by Sama34 — APPROVED** [review]

File: global.php (hunk 1 above)
Comment (Eldenroot) [reply global.php:650]: Maybe I should revert it back if it is necessary. I wanted to solve the mentioned issue in a different way but then I found easier way without a lot of changes... this should be fine, but no problem to change it to the default code.

Comment (Sama34) [reply global.php:650]: I don't mind, but it would probably be better to keep has it was? @mybb/developers

Comment (Ben-MyBB) [reply global.php:650]: If it's no longer being used to resolve the issue we should probably revert this part.

**Comment by Ben-MyBB** [comment]
@Eldenroot we'll need to bump the template here as well

**Comment by Eldenroot** [comment]
Done!

File: inc/languages/english/global.lang.php [hunk 2]
```diff
@@ -328,8 +328,9 @@
 $l['unapproved_attachments'] = "{1} unapproved attachments";
 $l['unread_report'] = "1 unread report";
 $l['unread_reports'] = "{1} unread reports";
-$l['pending_joinrequest'] = "Group Leader Notice: You have 1 pending group membership join request.";
-$l['pending_joinrequests'] = "Group Leader Notice: You have {1} pending group membership join requests.";
+$l['groupleader_notice'] = "Group Leader Notice:";
```
Comment (effone) [inline inc/languages/english/global.lang.php:331]: What about "Pending Group Leader Action(s)"?

Comment (Sama34) [reply inc/languages/english/global.lang.php:331]: How many _actions_ are there for group leaders?

**Review by Sama34 — APPROVED** [review]

**Comment by Eldenroot** [comment]
@Sama34 - I wanted to use same approach how we have for moderators notice... if it is not good, I can change it.

**Comment by Eldenroot** [comment]
I think this is done and can be merged :)

**Comment by Sama34** [comment]
I did test.

**Pull request merged**